docs/api/materials/ShadowMaterial.html

```diff
@@ -34,7 +34,11 @@
 
 		<h2>Constructor</h2>
 
-		<h3>[name]( )</h3>
+		<h3>[name]( [page:Object parameters] )</h3>
+		<div>
+			[page:Object parameters] - (optional) an object with one or more properties defining the material's appearance.
+			Any property of the material (including any property inherited from [page:Material] and [page:ShaderMaterial]) can be passed in here.<br /><br />
```

Comment:
Or maybe we can initially write a more generic sentence and do the detail documentation later...